cookbook/migration-v1-v2.md

```diff
@@ -138,23 +135,23 @@ This is due to the modernization of dist files to support native ESM modules in
   }
   ```
 
-## Devtools
+## Devtools {#devtools}
```

Comment:
I already translate `devtools` as `herramientas de desarrollo`, so we need to reach an agreement about that.

Reply:
I think that for the translation of the title it is applicable.
But for this content I think devtools is more understandable, what do you think?
![Screenshot 2023-01-08 at 14 28 58](https://user-images.githubusercontent.com/48701217/211217654-25c82faa-5c85-4d54-abcf-1cd43fa137de.png)

Reply:
Since _Vue Devtools_ is a thing I also believe it can stay without translation, but what about translate only the word `devtools`. For example in the file **getting-started.md** there was this sentence:

> This will also add devtools support.

Which I translate into:

> Esto también añade soporte para las herramientas de desarrollo.

I feel that translate it in these cases doesn't make the text less understandable.

Reply:
I don't think I explained it correctly, but I was referring to the previous code snippet. (Screenshot)
––––
Vue Devtools v5 or Vue Devtools v6
––––
and I do agree with you that the translation should be: "herramientas de desarrollo"